### DISCUSSION TITLE
Rename VpnHood (any our app block all processes with name VPN)

### DISCUSSION THREAD

**Discussion by Onair-santa** [vpnhood/VpnHood#353] [Q&A]
![image](https://github.com/vpnhood/VpnHood/assets/42511409/7997ba00-6661-42a4-854f-56396d294471)
Can you help me and rename all processes with VPN? may be only Hood or Edge or EdgeHood ...
In my fork.
Thnx for good job. Very very good vpn 
+++

**Comment by trudyhood** [comment]
Search and replace it all 😃 
Unfortunately we don't offer source code support. Happy Coding!
